decaffeinate-simple.sh

```diff
@@ -0,0 +1,19 @@
+set -ex
```

Comment:
Are we keeping this simple version and the full version?

Reply:
no, i think we can get rid of it now. will do